calendar_v17/odoo_office365_calendar/models/calendar_event.py

```
@@ -0,0 +1,651 @@
+
+                    recurrence_data = {
+                        "Pattern": pattern,
+                        "Range": range_data
+                    }
+                if rec.o365_event_id:
+                    office365.makerequest("PATCH", f"events/{rec.o365_event_id}", value)
+                else:
+                    ofsresponse = office365.makerequest("POST", "events", value)
+                    ofsresponse = json.loads(ofsresponse)
+                    if ofsresponse and ofsresponse.get("id"):
+                        rec.o365_event_id = ofsresponse.get("id")
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)